bluesky_queueserver/manager/manager.py

```diff
@@ -19,13 +19,13 @@
 """
 #  The following plans that can be used to test the syste
 
-http POST http://localhost:8080/add_to_queue plan:='{"name":"count", "args":[["det1", "det2"]]}'
+http POST http://localhost:60610/add_to_queue plan:='{"name":"count", "args":[["det1", "det2"]]}'
 
 # This is the slowly running plan (convenient to test pausing)
-http POST http://localhost:8080/add_to_queue plan:='{"name":"count", "args":[["det1", "det2"]],
+http POST http://localhost:60610/add_to_queue plan:='{"name":"count", "args":[["det1", "det2"]],
 "kwargs":{"num":10, "delay":1}}'
 
-http POST http://localhost:8080/add_to_queue plan:='{"name":"scan", "args":[["det1", "det2"], "motor", -1, 1, 10]}'
+http POST http://localhost:60610/add_to_queue plan:='{"name":"scan", "args":[["det1", "det2"], "motor", -1, 1, 10]}'
```

Comment:
In #63 I proposed to remove these examples as they are not valid anymore. Maybe it can be handled better here.

Reply:
Removed

Reply:
Thanks! I'll close #63.